.travis.yml

```diff
@@ -28,3 +28,12 @@ matrix:
     - rvm: 2.5
       gemfile: spec/support/gemfiles/Gemfile.rails-5.1.x
       env: DB=postgres
+    - rvm: 2.5
+      gemfile: spec/support/gemfiles/Gemfile.rails-5.2.x
+      env: DB=mysql
+    - rvm: 2.5
+      gemfile: spec/support/gemfiles/Gemfile.rails-5.2.x
+      env: DB=sqlite
+    - rvm: 2.5
+      gemfile: spec/support/gemfiles/Gemfile.rails-5.2.x
+      env: DB=postgres
```

Comment:
👏 

To reduce build times, could we remove these two from the matrix:

```yaml
     - rvm: 2.5
       gemfile: spec/support/gemfiles/Gemfile.rails-5.1.x
       env: DB=sqlite

     - rvm: 2.5
       gemfile: spec/support/gemfiles/Gemfile.rails-5.1.x
       env: DB=postgres
```

Reply:
@orien I would love to leave it there to cover these 2 adapters.

Reply:
They'll still be covered by the Rails 5.2 entries you've just added.